app/styles/_core.less

```diff
@@ -532,6 +539,38 @@ label.checkbox {
   }
 }
 
+.osc-secrets-form {
+  .osc-secret-actions {
+    font-size: 11px;
```

Comment:
Yeah, I was suggesting we make them all the default font size and left-align the text with the content rather than indenting. We should make it look like the "Add environment variable" action for consistency with other places in our UI.